addons/accessibility/fnc_showHints.sqf

```diff
@@ -0,0 +1,18 @@
+#include "script_component.hpp"
+
+for "_i" from (count GVAR(hints) - 1) to 0 step -1 do {
+    private _hint = GVAR(hints) select _i;
+    _hint params ["_text", "_source", "_expiresAt"];
+
+    if (diag_tickTime > _expiresAt) then {
```

Comment:
Probably not savegame compatible, as diag_tickTime (unlike CBA_missionTime) is not strictly monotonically increasing.

Reply:
Also not pause compatible, runs through singleplayer pause screen